pom.xml

```diff
@@ -46,6 +46,7 @@
         <glassfish-json.version>1.0.2</glassfish-json.version>
         <guava.version>19.0</guava.version>
         <hamcrest.date.version>2.0.4</hamcrest.date.version>
+        <hamcrest.regex.version>1.0.1</hamcrest.regex.version>
```

Comment:
I know this is being picky, but could you put it in correct alphabetical order please :) - Happy for you to do a straight amend or interactive rebase to squash back to 1 commit, then I'll merge it.